develop/devguide/Connector/ImpactOfProtocolChanges/TableParameterChanges/ChangeDisplayKey.md

```diff
@@ -0,0 +1,92 @@
+---
+uid: ChangeDisplayKey
+---
+
+# Change display key
+
+Changing the display key of a table is considered a major change.
+
+## Impact
+
+This has the same impact as changing a parameter description. The following DataMiner features can be impacted if used:
+
+- Filters
+- Automation scripts
+- Visio files
+- Reports
+- Web API
+
+*DIS MCC*
+
+|            | Error message | Description                                                                                                        |
+|------------|---------------|--------------------------------------------------------------------------------------------------------------------|
+| **2.39.4** | FormatChanged | Table display key was changed from {oldSyntax} '{oldFormat}' to {newSyntax} '{newFormat}'. Table PID '{tablePid}'. |
+| **2.39.5** | FormatRemoved | Table display key previously defined via '{oldSyntax}' was removed. Table PID '{tablePid}'.                        |
+
+## Workarounds
+
+### Display key column exists
+
+If a display key already exists, and you want to change its format, the following routine should be implemented to avoid impact by default:
+
+- R/W (standalone) discreet parameter to select the naming format
+
+- Default value should be the original format (no impact for existing users)
+
+- Ref. SLC SDF Naming Format Compatibility
+
+  - Version PE.0.0.1
+  - Located on SVN: <https://svn.skyline.be/svn/SystemDevelopmentFeature/SLC SDF Naming Format Compatibility/>
+
+> [!NOTE]
+> It is possible to implement a workflow to change the display key format on element creation (when the default display key is not the desired one). This will avoid a manual action after every element creation. You can implement this with a correlation rule and an Automation script.
+
+### Display key column does not exist (primary key is the current display key)
+
+- Copy the description of the primary key column to the new display key column.
+
+- Give the primary key column a new description and hide this column (set width to 0).
+
+- Display the new display key column in the original position of the primary key column.
+
+- The rest of workaround stays the same (see SLC SDF Naming Format Compatibility):
```

Comment:
Same problem as above: where can "SLC SDF Naming Format Compatibility" be found?